packages/client/components/NewMeetingCarousel.tsx

```diff
@@ -0,0 +1,148 @@
+import styled from '@emotion/styled'
+import React from 'react'
+import {FreeMode, Mousewheel} from 'swiper'
+import 'swiper/css'
+import 'swiper/css/free-mode'
+import 'swiper/css/mousewheel'
+import {Swiper, SwiperSlide} from 'swiper/react'
+import action from '../../../static/images/illustrations/action.png'
+import retrospective from '../../../static/images/illustrations/retrospective.png'
+import poker from '../../../static/images/illustrations/sprintPoker.png'
+import teamPrompt from '../../../static/images/illustrations/teamPrompt.png'
+import {PALETTE} from '../styles/paletteV3'
+import {BezierCurve, Breakpoint, NewMeeting} from '../types/constEnums'
+import {MeetingTypeEnum} from '../__generated__/NewMeetingQuery.graphql'
+
+const MEDIA_QUERY_FUZZY_TABLET = `@media screen and (max-width: ${Breakpoint.FUZZY_TABLET}px)`
+
+const Container = styled('div')({
+  '.swiper-wrapper': {
+    alignItems: 'center',
+    padding: '20px 0'
+  },
+  marginTop: -12
+})
+
+const Title = styled('div')<{isActive: boolean}>(({isActive}) => ({
+  color: isActive ? PALETTE.WHITE : PALETTE.SLATE_900,
+  fontSize: 20,
+  fontWeight: 600
+}))
+
+const Description = styled('div')<{isActive: boolean}>(({isActive}) => ({
+  display: isActive ? 'block' : 'none',
+  overflow: 'hidden',
+  color: PALETTE.WHITE,
+  fontSize: 12,
+  [MEDIA_QUERY_FUZZY_TABLET]: {
+    fontSize: 11
```

Comment:
+1 this seems to be pretty small from accessibility pov, can we stick to at least 12?

Reply:
Made 12px